superset/sql_lab.py

```diff
@@ -400,7 +400,7 @@ def execute_sql_statements(
                 )
             cache_timeout = database.cache_timeout
             if cache_timeout is None:
-                cache_timeout = config.get("CACHE_DEFAULT_TIMEOUT", 0)
+                cache_timeout = config["CACHE_DEFAULT_TIMEOUT"]
```

Comment:
I wonder what the point of defaulting to 0 was here. I can only assume it is a relic from long ago.

Reply:
It must be a relic, as it is set in config.py.